perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/bevpool_plugin.hpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2021-2023, NVIDIA CORPORATION. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef AUTOWARE__TENSORRT_BEVDET__BEVPOOL_PLUGIN_HPP_
+#define AUTOWARE__TENSORRT_BEVDET__BEVPOOL_PLUGIN_HPP_
+
+#include <NvInfer.h>
+#include <unistd.h>
+
+#include <cmath>
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+namespace
+{
+static const char * BEVPOOL_PLUGIN_NAME{"BEVPool"};
+static const char * BEVPOOL_PLUGIN_VERSION{"1"};
+}  // namespace
+
+namespace nvinfer1
+{
+class BEVPoolPlugin : public IPluginV2DynamicExt
+{
+private:
+  const std::string name_;
```

Comment:
Consider making `name_` a const member to reflect that it's not modified after construction. This enhances code clarity and prevents accidental modifications.